### PR TITLE
KOGITO-3133 Expose nodeInstanceId in Job details Graphql

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Review by tiagodolphine — APPROVED** [review]
LGTM

File: data-index/data-index-service/data-index-service-common/src/test/java/org/kie/kogito/index/service/AbstractIndexingServiceIT.java [hunk 1]
```diff
@@ -55,6 +55,7 @@
 import static org.hamcrest.Matchers.emptyOrNullString;
 import static org.hamcrest.Matchers.hasItems;
 import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.notNullValue;
```
Comment (jstastny-cz) [inline data-index/data-index-service/data-index-service-common/src/test/java/org/kie/kogito/index/service/AbstractIndexingServiceIT.java]: is this actually used?

**Review by jstastny-cz — COMMENTED** [review]
Just wondering if there's not more places where to put tests, especially for the marshaller change.

**Review by nmirasch — APPROVED** [review]

**Review by Sara4994 — APPROVED** [review]

File: data-index/data-index-service/data-index-service-common/src/test/java/org/kie/kogito/index/service/AbstractIndexingServiceIT.java (hunk 1 above)
Comment (cristianonicolai) [reply data-index/data-index-service/data-index-service-common/src/test/java/org/kie/kogito/index/service/AbstractIndexingServiceIT.java]: good catch, removed.

**Review by jstastny-cz — APPROVED** [review]

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Pull request merged**